CONTRIBUTING.md

```diff
@@ -176,7 +176,7 @@ For a new files : new_file_create.dart
 ```
 
 ```
-For a new Classes : class MyClasse
+For new class: title-presentation
```

Comment:
class: TitlePresentation

components in the template in yaki-admin: title-presentation